docs/src/bootstrap.md

```diff
@@ -54,6 +54,10 @@ create a new cluster based on another one (either offline or online) and can be
 used to spin up replica clusters. They both rely on the definition of external
 clusters.
 
+Given the several possibilities, methods, and combinations that the
+CloudNativePG operator provides in terms of backup and recovery, please refer
+to the ["Recovery" section](recovery.md).
```

Comment:
```suggestion
to the ["Recovery" section](recovery.md) for guidance for each method.
```

Reply:
Just a suggestion. Haven't replicated it yet to the other locations of this text but...